scripts/pre_commit_linter.py

```diff
@@ -69,7 +69,7 @@
 
 _PARSER = argparse.ArgumentParser()
 _EXCLUSIVE_GROUP = _PARSER.add_mutually_exclusive_group()
-_EXCLUSIVE_GROUP.add_argument(
```

Comment:
Why this change? Can you please explain?

Reply:
Hi @apb7, I added this such that the user can use --mode and --path args at the same time, as I think that when --files args will be used there is not much use of --mode flag(as there will be very few files to lint usually 1-3), therefore, I made it exclusive.
What do you suggest?

Reply:
When we will use --path arg we can use the -v but when we will use --files we can't.

Reply:
Why? `--flies` flag can have multiple files as arguments, so the flag should work for this case too.